src/rest_framework_dso/filters.py

```diff
@@ -642,9 +642,11 @@ def get_ordering(self, request, queryset, view):
         if ordering is None:
             return ordering
 
-        # convert to snake_case, preserving `-` if needed
-        correct_ordering = ["-".join([to_snake_case(y) for y in x.split("-")]) for x in ordering]
-        return correct_ordering
+        # Convert identifiers to snake_case, preserving `-` (descending sort).
+        return [
```

Comment:
Part slicing is dangerous. Say we will have a field ('test-data'). This code will break things.

Maybe following will do?:
```
        correct_ordering = ["-".join([to_snake_case(y) for y in x.split("-") if y]) for x in ordering]
````

Reply:
OR: update `to_snake_case` to ignore empty strings? Probably not such a good idea, but worth discussing

Reply:
Are such field names allowed at all? If so, to_snake_case is broken, because it returns test_data, so we have a much bigger problem.

Reply:
No, they are not. But again, slicing is no good. It won't save us if someboy uses `--name` as sort parameter.

Reply:
Oh, i see now that my brainfart code is bad idea :)  as it will drop `-` from `-name` filter.